workspaces/ui-v2/src/pages/changelog/ChangelogEndpointRootPage.tsx

```diff
@@ -49,20 +50,31 @@ const ChangelogRootComponent: FC<
     batchId: string;
   }>
 > = ({ match }) => {
-  const { endpoints, loading } = useEndpoints();
+  const endpointsState = useAppSelector((state) => state.endpoints.data);
+
+  const dispatch = useAppDispatch();
+  const spectacle = useSpectacleContext();
+  useEffect(() => {
+    // the typing is correct, we just arent setting it correctly above
+    // TODO remove ts-ignore
+    // @ts-ignore
+    dispatch(endpointActions.fetchEndpoints({ spectacle }));
+  }, [dispatch, spectacle]);
```

Comment:
This is what interacting with the store would look like - I'm probably going to pull these into separate functions such that we can reuse this easier but the core interactions with redux is here